src/EventStore.Core/TransactionLog/Chunks/TFChunk/PosMap.cs

```diff
@@ -1,7 +1,11 @@
+using System;
+using System.Diagnostics;
 using System.IO;
+using DotNext.Buffers;
+using DotNext.Buffers.Binary;
 
 namespace EventStore.Core.TransactionLog.Chunks.TFChunk {
-	public struct PosMap {
+	public struct PosMap : IBinaryFormattable<PosMap> {
```

Comment:
i don't think the new code in this file is used, should it be?

Reply:
That was my attempt to migrate away from BinaryReader/BinaryWriter using this new abstraction. Now fixed in the following way: overloads with `BinaryReader`/`BinaryWriter` just delegate calls to Span-based versions.